install/kubernetes/istio-auth.yaml

```diff
@@ -273,6 +273,7 @@ spec:
         args:
           - --configStoreURL=fs:///etc/opt/mixer/configroot
           - --configStore2URL=k8s://
+          - --configDefaultNamespace=istio-system
           - --logtostderr
           - -v
           - "4"
```

Comment:
is it worth turning down logging? doesn't v4 cause unnecessary performance burden?

Reply:
probably. but let's fix that in a different PR.

Reply:
You sure? One of our benchmarks showed a significant slowdown in perf when we turned on logging. But that was for pilot. I am just extrapolating.

Reply:
Mixer logs a lot.

Reply:
My concern is just that we might want to wait to turn down logging once things have settled a bit more and we are closer to release.  Prod deployments of Mixer should probably set v=0.

Reply:
True - I guess lets keep it verbose till we get closer to the final 0.2 cut.

Reply:
can we change the flag default value too to be `istio-system`